### ISSUE TITLE
Atom-Typescript error: Debug Failure. Expected C:/Users/walla/OneDrive/Documents/Angular/tsconfig...

### ISSUE THREAD

**Issue by wallacesfos** [TypeStrong/atom-typescript#1579]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.55.0 x64
**Electron**: 6.1.12
**OS**: Unknown Windows version
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package 14.1.2

### Stack Trace

Atom-Typescript error: Debug Failure. Expected C:/Users/walla/OneDrive/Documents/Angular/tsconfig.json === C:\Users\walla\OneDrive\Documents\Angular\tsconfig.json. 

```
At Error: Debug Failure. Expected C:/Users/walla/OneDrive/Documents/Angular/tsconfig.json === C:\Users\walla\OneDrive\Documents\Angular\tsconfig.json. 

Error: Debug Failure. Expected C:/Users/walla/OneDrive/Documents/Angular/tsconfig.json === C:\Users\walla\OneDrive\Documents\Angular\tsconfig.json. 
    at attachFileToDiagnostic (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:18897:18)
    at Object.attachFileToDiagnostics (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:18929:42)
    at Object.parseJsonText (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:29058:46)
    at Object.parseJsonText (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:28804:23)
    at parseConfigFileTextToJson (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:37574:33)
    at Object.readConfigFile (/packages/atom-typescript/node_modules/typescript/lib/typescript.js:37565:48)
    at Ae (/packages/atom-typescript/dist/main.js:1:102508)
    at Ie (/packages/atom-typescript/dist/main.js:1:102213)
    at hu.readConfigFile (/packages/atom-typescript/dist/main.js:1:294469)
    at /packages/atom-typescript/dist/main.js:1:294342
    at Function.simpleDispatch (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:11:1190181)
    at Emitter.emit (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:11:1191622)
    at File.handleNativeChangeEvent (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:1137061)
    at PathWatcher.<anonymous> (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:1137754)
    at PathWatcher.onChange (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:11411)
    at PathWatcher.onChange (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:11867)
    at Function.simpleDispatch (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:11:1190181)
    at Emitter.emit (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:11:1191622)
    at HandleWatcher.onEvent (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:10119)
    at ~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:14:12613
```

### Commands

```
     -0:57.3.0 editor:newline (input.hidden-input)
     -0:56.7.0 core:backspace (input.hidden-input)
     -0:56.4.0 snippets:next-tab-stop (input.hidden-input)
     -0:56.4.0 snippets:expand (input.hidden-input)
     -0:56.4.0 editor:indent (input.hidden-input)
  5x -0:53.5.0 core:backspace (input.hidden-input)
     -0:45.8.0 core:move-right (input.hidden-input)
  2x -0:43 core:backspace (input.hidden-input)
     -0:41.2.0 editor:newline (input.hidden-input)
     -0:28.6.0 core:move-right (input.hidden-input)
     -0:27.9.0 core:backspace (input.hidden-input)
     -0:26.5.0 editor:newline (input.hidden-input)
  3x -0:21.2.0 core:backspace (input.hidden-input)
     -0:16.5.0 core:move-right (input.hidden-input)
     -0:15.4.0 core:backspace (input.hidden-input)
     -0:11.1.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
atom-typescript 14.1.2 
```

**Comment by lierdakil** [comment]
I've released v14.2.0 which updates the bundled typescript version. Hopefully this issue is fixed there.

**Comment by lierdakil** [comment]
I'll close this as a duplicate of #1577
